src/pages/Books.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Books({ book }) {
+  return (
+    <div className="book-con">
+      <h3>
+        {book.genre}
+      </h3>
+      <h1>
+        {book.title}
+      </h1>
+      <p>
+        {book.author}
+      </p>
+      <ul className="navs">
+        <li>
+          <button type="button">
+            Comments
+          </button>
+        </li>
+        <li>
+          <button type="button">
+            Remove
+          </button>
+        </li>
+        <li>
+          <button type="button">
+            Edit
+          </button>
+        </li>
+      </ul>
+    </div>
+  );
+}
```

Comment:
- [ ] The `Books` or `BookList` component should make use of another `Book` component by maping over an array of books then passing the book data as a prop to render the title and the author (also the remove button). This will make it a bit easier when invoking redux action in the next milestones.

Reply:
This issue is well resolved! ✔️